exec/java-exec/src/main/java/org/apache/drill/exec/record/RecordBatch.java

```diff
@@ -115,6 +115,14 @@
      *   This value will be returned only after {@link #OK_NEW_SCHEMA} has been
      *   returned at least once (not necessarily <em>immediately</em> after).
      * </p>
+     * <p>
+     *   Also after a RecordBatch returns NONE a RecordBatch should:
+     *   <ul>
+     *     <li>Contain the last valid schema seen by the operator.</li>
+     *     <li>Contain an empty VectorContainer with empty columns corresponding to the last valid schema.</li>
```

Comment:
Is `empty` required ?

Reply:
Fixed thanks for catching and thanks for the review!